Problem on update package properties pipeline

While executing the pipeline I got the error below
/usr/bin/python: Error while finding module specification for 'packaging_automation.update_package_properties' (ModuleNotFoundError: No module named 'packaging_automation')
Error: Process completed with exit code 1.
tools prefix is missing in command. This prefix should be added

--- a/.github/workflows/build-package.yml
+++ b/.github/workflows/build-package.yml
@@ -32,7 +32,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.0 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
 
       - name: Install package dependencies
         run: sudo apt install libcurl4-openssl-dev libssl-dev

--- a/.github/workflows/update_package_properties.yml
+++ b/.github/workflows/update_package_properties.yml
@@ -30,17 +30,23 @@ jobs:
         run: sudo apt install libcurl4-openssl-dev libssl-dev
 
       - name: Clone tools branch
-        run: git clone -b v0.8.0 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b issue-149 --depth=1  https://github.com/citusdata/tools.git tools
+
+      - name: Set git name and email
+        run: |
+          git config --global user.email "${{ github.event.inputs.microsoft_email }}" && \
+          git config --global user.name "${{ github.event.inputs.name }}"
 
       - name: Install python requirements
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
       - name: Update package properties
         run: |
-          python -m packaging_automation.update_package_properties \
+          python -m tools.packaging_automation.update_package_properties \
           --gh_token="${GH_TOKEN}" \
           --prj_name "${PRJ_NAME}" \
           --tag_name ${{ github.event.inputs.tag_name }} \
           --email ${{ github.event.inputs.microsoft_email }} \
-          --name ${{ github.event.inputs.name }} --pipeline \
+          --name "${{ github.event.inputs.name }}" \
+          --pipeline \
           --exec_path "$(pwd)"

--- a/.github/workflows/update_package_properties.yml
+++ b/.github/workflows/update_package_properties.yml
@@ -30,7 +30,7 @@ jobs:
         run: sudo apt install libcurl4-openssl-dev libssl-dev
 
       - name: Clone tools branch
-        run: git clone -b issue-149 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
 
       - name: Set git name and email
         run: |

